flink-connectors/flink-connector-kafka-base/src/main/java/org/apache/flink/table/descriptors/KafkaValidator.java

```diff
@@ -38,7 +38,6 @@
 public class KafkaValidator extends ConnectorDescriptorValidator {
 
 	public static final String CONNECTOR_TYPE_VALUE_KAFKA = "kafka";
-	public static final String CONNECTOR_VERSION_VALUE_08 = "0.8";
```

Comment:
`ConnectorDescriptorValidator.java` contains a reference to Kafka 0.8 in line 47.

Reply:
`KafkaShortRetentionTestBase.java` contains in line 253 Kafka 0.8 specific code. In line 254 the same class contains Kafka 0.9 specific code.